test/testdrive/mzcompose.yml

```diff
@@ -66,6 +139,26 @@ services:
     - SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS=PLAINTEXT://kafka:9092
     - SCHEMA_REGISTRY_HOST_NAME=localhost
     depends_on: [kafka, zookeeper]
+
+  localstack:
+    image: localstack/localstack:0.12.5
+    ports:
+      - "4566"
+      - "4571"
+      - "${LOCALSTACK_WEB_UI-8080}"
+    environment:
+      - SERVICES=${SERVICES- }
+      - DEBUG=${DEBUG- }
+      - DATA_DIR=${DATA_DIR- }
+      - PORT_WEB_UI=${PORT_WEB_UI- }
+      - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
+      - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
+      - DOCKER_HOST=unix:///var/run/docker.sock
+      - HOST_TMP_FOLDER=${TMPDIR}
+    volumes:
+      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
+      - "/var/run/docker.sock:/var/run/docker.sock"
```

Comment:
This looks like it's copy pasted from the localstack docs, but some of these settings make me nervous. I tested that it pares down nicely to:

```
  localstack:
    image: localstack/localstack:0.12.5
    environment:
    - SERVICES=kinesis,s3
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```